app/services/github/repositories/sync.rb

```diff
@@ -32,6 +27,33 @@ def call
       def repositories
         @repositories ||= Repository.all
       end
+
+      def repository_owner_and_name(repository_link)
+        repository_link.split('/')[-2..-1]
+      end
+
+      def update_repository_languages(repository)
+        repository_owner, repository_name = repository_owner_and_name(repository.link)
+        request = client.repos.languages(repository_owner, repository_name)
+
+        if request.success?
+          languages = request.body.keys[0..MAX_LANGUAGES_NUMBER-1].join(', ')
+
+          repository.update!(language: languages)
+        end
+      end
+
+      def update_repository_stats(repository)
+        repository_owner, repository_name = repository_owner_and_name(repository.link)
+        request = client.repos.get(repository_owner, repository_name)
+
+        if request.success?
+          repository.update!(
+            issues: request.body['open_issues_count'],
+            stars: request.body['stargazers_count']
+          )
+        end
+      end
```

Comment:
can you organize the order of those methods based on the calling order? first is `update_repository_languages`, then `update_repository_stats` for instance. I think this might increase the readability a little bit. What do you think?